packages/remirror__extension-react-component/src/portals/react-portals.tsx

```diff
@@ -36,13 +37,13 @@ export interface RemirrorPortalsProps {
  * passed through to the `RemirrorPortals` component.
  */
 export function usePortals(portalContainer: PortalContainer): Array<[HTMLElement, MountedPortal]> {
-  const [portals, setPortals] = useState(() => [...portalContainer.portals.entries()]);
+  const [portals, setPortals] = useState(() => Array.from(portalContainer.portals.entries()));
```

Comment:
Good spot @strdr4605! 

If you don't mind, could you disable each line, rather than the entire file?

```suggestion
  // eslint-disable-next-line unicorn/prefer-spread
  const [portals, setPortals] = useState(() => Array.from(portalContainer.portals.entries()));
```